app/Locale/Slovak.php

```diff
@@ -376,8 +370,8 @@ public function get_locale(): array
 			'LANG_TEXT' => 'Zmeniť jazyk Lychee na:',
 			'LANG_TITLE' => 'Zmena jazyka',
 
-			'CSS_TEXT' => 'Vlastné CSS:',
-			'CSS_TITLE' => 'CSS zmeniť',
+			'CSS_TEXT' => 'Personalize CSS:',
+			'CSS_TITLE' => 'Change CSS',
```

Comment:
I can't speak Slovene. But given the fact that you replaced perfectly fine German translations by the English original, I assume the same error happened here. Why?

Reply:
Mistake. Fixed [5d3a3ef](https://github.com/LycheeOrg/Lychee/pull/1539/commits/5d3a3eff43046bc0fb28645b66d6055346132b66).